### ISSUE TITLE
using gpd after segmentation

### ISSUE THREAD

**Issue by frk1993** [atenpas/gpd2#5]
Hi,
i want to use this algorithm for a bin picking problem. Before planning the grasp pose I am segmenting the objects in the bin. I want to apply gpd only to one segmented object area. Can I do this ? To do so I should pick out the segmented area from the point cloud and fit only the segmented are point cloud?

Best Regards

**Comment by atenpas** [comment]
Yes, you can do this in two ways. You can use [`Cloud::setSampleIndices`](https://github.com/atenpas/gpd2/blob/master/include/gpd/util/cloud.h#L338) to provide a list of point cloud indices at which to search for grasps. Or you can use [`Cloud::setSamples`](https://github.com/atenpas/gpd2/blob/master/include/gpd/util/cloud.h#L346) to provide a list of points (as a 3xn matrix). You can create the indices/points based on the segmented object area.
